docs/_docs/04_reference/04-config-blocks-and-attributes.md

```diff
@@ -96,6 +96,8 @@ The `terraform` block supports the following arguments:
   - Refer to [A note about using modules from the
     registry]({{site.baseurl}}/docs/getting-started/quick-start#a-note-about-using-modules-from-the-registry) for more
     information about using modules from the Terraform Registry with Terragrunt.
+  - VERSION can be a specific version number (e.g., `1.2.3`) or a version constraint (e.g., `>= 1.2.3, <=3.2.1`) following syntax
+    defined in the [Terraform documentation](https://developer.hashicorp.com/terraform/language/expressions/version-constraints).
```

Comment:
```suggestion
    defined in the [OpenTofu documentation](https://opentofu.org/docs/language/expressions/version-constraints/).
```

We always attempt to default to OpenTofu documentation. Please make sure there isn't anything missed in the rest of the pull request that prefers Terraform documentation over OpenTofu.